utils/useDatabase.js

```diff
@@ -1,7 +1,10 @@
-import { supabaseAdmin } from './initSupabaseAdmin';
-import { stripe } from './initStripe';
+import { supabaseAdmin } from './supabase-admin';
+import { stripe } from './stripe';
 import { toDateTime } from './helpers';
 
+// This entire file should be removed and moved to supabase-admin
+// It's not a react hook, so it shouldn't have useDatabase format
+// It should also properly catch and throw errors
```

Comment:
@thorwebdev would you mind looking at this? 🙏 

Basically, I'd like to keep the functions that access Supabase inside a single file where you'd likely never pull the supabase instance directly. Instead, you write a helper function to perform the action. Then you have self-documenting function names 😄

Reply:
I'm curious about this @leerob - do you mean it would be something like this?

```js
export const upsert = (table, data) => {
  supabase.from(table).insert(data, { upsert: true })
}
```

? 

Can you show the pattern you're thinking of for self-documenting function names?

Reply:
Here's an example. Before, there was a comment.

```js
  // Load all active products and prices at build time.
  const { data: products, error } = await supabase
    .from('products')
    .select('*, prices(*)')
    .eq('active', true)
    .eq('prices.active', true)
    .order('metadata->index')
    .order('unit_amount', { foreignTable: 'prices' });
  if (error) console.log(error.message);
```
  
Instead, let's just move all of this logic to the "library" level. Remove the comment, because the function name self-documents what it's doing.
  
```js
const products = await getActiveProductsWithPrices();
```